test/src/main/scala/org/scalatra/test/Client.scala

```diff
@@ -94,10 +94,14 @@ trait Client extends ImplicitConversions {
     patch(uri, params)(f)
   def patch[A](uri: String, params: Iterable[(String, String)])(f: => A): A =
     patch(uri, params, Map[String, String]())(f)
-  def patch[A](uri: String, params: Iterable[(String, String)], headers: Iterable[(String, String)])(f: => A): A =
+  def patch[A](uri: String, params: Iterable[(String, String)], headers: Map[String, String])(f: => A): A =
```

Comment:
This one looks a change for a different purpose (consistency with POST/PUT ..). Although that is a breaking change but if there is no objection from others, I think it's ok to merge for 2.5.1.